src/test/java/org/folio/circulation/support/JsonPropertyFetcherTest.java

```diff
@@ -0,0 +1,38 @@
+package org.folio.circulation.support;
+
+import static org.folio.circulation.support.JsonPropertyFetcher.getDateTimePropertyByPath;
+import static org.folio.circulation.support.JsonPropertyWriter.writeByPath;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.nullValue;
+
+import org.joda.time.DateTime;
+import org.junit.Test;
+
+import io.vertx.core.json.JsonObject;
+
+public class JsonPropertyFetcherTest {
+
+  @Test
+  public void shouldReturnNullIfSomeObjectsNotExistInPath() {
```

Comment:
Is there another scenario where all of the objects in the path exist however the property does not?

Reply:
I've added one, thanks.